### DISCUSSION TITLE
[Feature] apply additional user-defined WHERE condition to assets in the timeline view

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6277] [Feature Request]
### The feature

I would like to apply user defined WHERE conditions, which act as additional filters for assets, shown in the timeline.

It is fine to be able to query the assets table in the database, but the results can't be shown in the immich frontend. If we could apply additional filters in the GUI to the timeline, we could flexibly filter assets, until specific filters are implemented.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #3477
